.eslintrc.json

```diff
@@ -23,6 +23,8 @@
     "ignorePatterns": [
       "dist/",
       "build/",
-      "luxon.js", "**/assets/*.js"
+      "luxon.js", "**/assets/*.js",
+      "server.js",
+      "webpack.config.js"
```

Comment:
### Spoiler Alert 

- [  ]  Please next don't add anything to the **`eslintrc`** file.